### DISCUSSION TITLE
[博客推荐] 工程师如何对待开源：一个老工程师的肺腑之言

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#33] [1️⃣ 新鲜事]
谭中意老师是一个在知名科技企业内从事开源相关工作超过 20 年的工程师，亲身经历或者亲眼目睹很多工程师对待开源软件的优秀实践，也看到了很多 Bad Cases，所以把自己的一些心得体会写在这里，供工程师进行参考，希望能帮助工程师更好的成长。推荐大家阅读~

https://my.oschina.net/u/3742410/blog/5526206

**Comment by zhicheng-ning** [comment]
获益匪浅 :smile:

**Comment by will-ww** [comment]
下周二（5月31日）谭中意老师会来我们的开源课程上做分享，大家有什么想问的问题，可以这里随时交流~
